src/raster/rasterband.rs

```diff
@@ -689,7 +656,7 @@ impl<'a> RasterBand<'a> {
                 window.1 as c_int,
                 window_size.0 as c_int,
                 window_size.1 as c_int,
-                buffer.data.as_ptr() as GDALRasterBandH,
+                buffer.data.as_ptr() as *mut c_void,
```

Comment:
The `as GDARasterBandH` was semantically incorrect.

Reply:
:scream_cat: I didn't even notice that.